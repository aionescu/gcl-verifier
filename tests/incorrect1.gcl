incorrect1(x: Int) -> y: Int {
  assert x == 7;
}
