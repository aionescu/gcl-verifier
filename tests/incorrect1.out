"Your program is not correct"